engine/client/ref_common.c

```diff
@@ -484,13 +485,22 @@ void R_Shutdown( void )
 
 void R_GetRendererName( char *dest, size_t size, const char *refdll )
 {
-	Q_snprintf( dest, size, "%sref_%s.%s",
+#ifdef XASH_INTERNAL_GAMELIBS
+    Q_snprintf( dest, size, "%sref_%s%c%s",
 #ifdef OS_LIB_PREFIX
-		OS_LIB_PREFIX,
+        OS_LIB_PREFIX,
+#else
+        "",
+#endif
+#ifndef NO_LIB_EXT
```

Comment:
NO_LIB_EXT is useless. iOS should use XASH_INTERNAL_GAMELIBS.

Also, check code style. You can find more info in https://github.com/FWGS/xash3d-fwgs/blob/master/CONTRIBUTING.md#code-style